src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -445,5 +448,21 @@ private void ThrowIfInvalidAccess()
                     "Multithreaded execution was detected. This can happen if the orchestrator function previously resumed from an unsupported async callback.");
             }
         }
+
+        public override void AddDeferredTask(Func<Task> function)

```

Comment:
Can these two methods be `internal` instead of `public`? I don't think we want to expose this externally.